can-construct-super.js

```diff
@@ -46,7 +47,7 @@ var isFunction = canIsFunction,
 Construct._defineProperty = function(addTo, base, name, descriptor) {
 	var _super = Object.getOwnPropertyDescriptor(base, name);
 	if(_super) {
-		canEach(getset, function (method) {
+		canReflect.eachKey(getset, function (method) {
```

Comment:
Because `getset` is an array, I think `canReflect.each` would be better here.

Reply:
Thanks, I had doubts about it, it's fixed!